graphql/operations.graphql

```diff
@@ -222,12 +222,24 @@ query CohortsList(
   $where: CohortWhere
   $options: CohortOptions
   $cohortsConnectionWhere2: CohortWhere
+  $hasCohortCompleteCohortCompletesOptions2: CohortCompleteOptions
```

Comment:
For showing the latest `CohortComplete` data by `date` value. See [this comment](https://github.com/mskcc/smile-dashboard/pull/113/files#r1529137887) for extra clarification.